server/i18n/en/errors.json

```diff
@@ -102,6 +102,9 @@
   },
   "referenceNumber": { "empty": "You must enter a reference number" },
   "query": { "empty": "You must enter a query" },
-  "userId": { "empty": "You must choose a user to allocate the application to" },
+  "userId": {
+    "empty": "You must choose a user to allocate the application to",
+    "lackingQualifications": "This user lacks the necessary qualifications"
```

Comment:
I'm hoping this won't be a problem now we've ensured the right users show up, but worth having this anyway 👍

Reply:
Yeah I figure something could change in the API to make it more likely to happen and if not it won't hurt to have it